src/vos/tests/vts_pm.c

```diff
@@ -935,8 +941,8 @@ sgl_test(void **state)
 	recx[0].rx_nr = SM_BUF_LEN;
 
 	/* Fetch whole buffer */
-	rc = vos_obj_fetch(arg->ctx.tc_co_hdl, oid, epoch++, &dkey, 1, &iod,
-			   &sgl);
+	rc = vos_obj_fetch(arg->ctx.tc_co_hdl, oid, epoch++, &dkey, 1,
```

Comment:
these changes could be reverted since it's not actually changing anything now

Reply:
will clean up in a subsequent patch